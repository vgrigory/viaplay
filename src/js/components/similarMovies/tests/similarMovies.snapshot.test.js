import React from 'react';
import SimilarMovies from '../similarMovies';
import renderer from 'react-test-renderer';

it('snapsot test for SimilarMovies component', () => {
    let similarMovies = {
        movies: [{
            publicPath: 'titanic-1997',
            content: {
                images: {
                    boxart: {
                        url: 'some url'
                    }
                }
            }
        }, {
            publicPath: 'the-choice-2016',
            content: {
                images: {
                    boxart: {
                        url: 'some other url'
                    }
                }
            }
        }],
        title: 'Similar movies'
    };

    const tree = renderer.create(
        <SimilarMovies data={similarMovies} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
});