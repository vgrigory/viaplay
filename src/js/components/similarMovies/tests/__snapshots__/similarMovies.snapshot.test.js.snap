exports[`test snapsot test for SimilarMovies component 1`] = `
<div
  className="carousel slide"
  onMouseOut={[Function]}
  onMouseOver={[Function]}>
  <ol
    className="carousel-indicators">
    <li
      className="active"
      onClick={[Function]} />
     
  </ol>
  <div
    className="carousel-inner">
    <div
      className="item active">
      <div
        className="container">
        <div
          className="row">
          <div
            className="col-lg-2 col-md-2 col-sm-2 col-xs-4">
            <a
              href="/film/titanic-1997">
              <div
                className="thumbnail">
                <img
                  alt={undefined}
                  src="some url" />
              </div>
            </a>
          </div>
          <div
            className="col-lg-2 col-md-2 col-sm-2 col-xs-4">
            <a
              href="/film/the-choice-2016">
              <div
                className="thumbnail">
                <img
                  alt={undefined}
                  src="some other url" />
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <a
    className="carousel-control left"
    href="#"
    onClick={[Function]}
    role="button">
    <span
      className="glyphicon glyphicon-chevron-left" />
    <span
      className="sr-only">
      Previous
    </span>
  </a>
  <a
    className="carousel-control right"
    href="#"
    onClick={[Function]}
    role="button">
    <span
      className="glyphicon glyphicon-chevron-right" />
    <span
      className="sr-only">
      Next
    </span>
  </a>
</div>
`;
